lib/suspenders/app_builder.rb

```diff
@@ -79,7 +79,7 @@ def configure_smtp
       RUBY
 
       inject_into_file 'config/environments/production.rb', config,
-        :after => 'config.action_mailer.raise_delivery_errors = false'
+        after: 'config.action_mailer.raise_delivery_errors = false'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.